### PR TITLE
Rename copy user ID tooltip

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

Minor tooltip change for copying the user ID.

Follow-up from https://github.com/gitpod-io/gitpod/pull/17956. Cc @filiptronicek @loujaybee 

## How to test

Go to `/user/account` and try to copy the new user ID field. The tooltip should use a new label.

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - gt-rename-e12d262cb2</li>
	<li><b>🔗 URL</b> - <a href="https://gt-rename-e12d262cb2.preview.gitpod-dev.com/workspaces" target="_blank">gt-rename-e12d262cb2.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [X] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/user-settings/Account.tsx [hunk 1]
```diff
@@ -162,7 +162,7 @@ function ProfileInformation(props: {
                         <div className="flex flex-col space-y-2 mt-4">
                             <label className={"text-md font-semibold dark:text-gray-400 text-gray-600"}>User ID</label>
                             <p className={"text-sm text-gray-500 dark:text-gray-500"}>
-                                <InputWithCopy className="max-w-md w-32" value={props.user.id} tip="Copy Token" />
+                                <InputWithCopy className="max-w-md w-32" value={props.user.id} tip="Copy User ID" />
```
Comment (easyCZ) [inline components/dashboard/src/user-settings/Account.tsx:165]: What if we left it at just `Copy`?

**Review by easyCZ — APPROVED** [review]

File: components/dashboard/src/user-settings/Account.tsx (hunk 1 above)
Comment (filiptronicek) [reply components/dashboard/src/user-settings/Account.tsx:165]: I'm with Milan on this one, keepin' it simple.

Comment (filiptronicek) [inline components/dashboard/src/user-settings/Account.tsx:165]:
```suggestion
                                <InputWithCopy className="max-w-md w-32" value={props.user.id} tip="Copy" />
```

**Comment by gtsiolis** [comment]
Thanks for taking a look, @easyCZ & @filiptronicek. 🌮 

/unhold

**Pull request merged**